freestyle-logging/jvm/src/main/scala/loggingJVM.scala

```diff
@@ -19,35 +19,84 @@ package freestyle
 import cats.MonadError
 import freestyle.logging._
 import journal._
+import org.slf4j.LoggerFactory
 
 object loggingJVM {
 
   object implicits {
-    implicit def freeStyleLoggingHandler[M[_], C: Manifest](
+    implicit def freeStyleLoggingHandler[M[_]](
         implicit ME: MonadError[M, Throwable]): LoggingM.Handler[M] =
       new LoggingM.Handler[M] {
 
-        val logger = Logger[C]
+        val logger = Logger("")
```

Comment:
We can't provide that here because it always infers the handler name and not the class invoking it.
I purposedly added an empty string to avoid showing the same non nonsensical logging we have been seeing until now.